### ISSUE TITLE
pip package?

### ISSUE THREAD

**Issue by helloagain-dev** [ztp-at/RKSV#3]
Did you ever think about providing this as a pip package or just creating a setup.py?
It would make it very easy to include your verification code then.

**Comment by ztp-mino** [comment]
We did not up until now. A pip package might be interesting but right now the verification code is pretty tightly coupled with the UI stuff and I should probably refactor to separate them first...

So maybe later? Do you need to integrate the verification into your own project?

**Comment by helloagain-dev** [comment]
Yes. We want to do receipt verification and we are thinking about using your code.
For now I created a fork with a setup.py and add it via ``pip install -e git+https://github.com/helloagain-dev/RKSV.git@60760f6f9f0ff513be9ccb5b2f085724ea673c1c#egg=rksv``.

**Comment by ztp-mino** [comment]
Keep in mind that our code here is licensed under the AGPLv3. You'll need to license whatever you make and distribute with our code under the AGPLv3 as well and you need to distribute according to the AGPLv3.

**Comment by helloagain-dev** [comment]
If I use it on a server, where I do not distribute the code and I do not modify your code, except adding the setup.py (which is published in the fork) I am fine, right?
I only call your python code.
